### DISCUSSION TITLE
DCCEx CS is not detected

### DISCUSSION THREAD

**Discussion by slotequalizer** [DCC-EX/EX-T3-WiFi#123] [General]
My T3 is connected to my home network and I would like to connect my T3 directly to my DCCEx CS (AP).  I have put the CS IP address in the cs settings and the port to 2560 and the T3 is not finding it.  Not sure what to do.  I have tested the CS with a cel phone app and it works fine.

**Comment by alex-code** [comment]
Did you connect the T3 to the CS's own network?
If the CS is in AP mode you'll need to connect to that.

On the WiFi settings view you can touch the labels to show a virtual keyboard/keypad so you can enter the CS AP details.

**Reply by slotequalizer** [reply]
I tried to connect the T3 to the CS's own network.  I used 192.168.4.1 as the server and 2560 as the port.  That is what works on my android phone with the Engine Driver app.  I also tried server DCCEX_6ed7b0 and 2560 as port and had the same result.

![IMG_4432](https://github.com/user-attachments/assets/792abb33-9807-478f-85dd-52d61a9688d2)
![IMG_4433](https://github.com/user-attachments/assets/ac197753-08da-4f06-ac25-f57a708c226b)
![IMG_4431](https://github.com/user-attachments/assets/9248da57-be46-4a34-9714-a7c0723f00fd)

**Reply by alex-code** [reply]
The top **WiFi Settings** will need to be your CS's AP details.
So **SSID** would be DCCEX_xxxx and **Password** would be PASS_xxxx changing the xxxx for the CS specific bits.
Possibly `DCCEX_6ed7b0` from your pics.

**Reply by slotequalizer** [reply]
That was it!  So when I want to manage the locos I have to change that back to my home wifi settings or is there an easier way.  I'm very happy with the throttle either way thank you so much for all your hard work.  It's a great unit.  If I use my CS in STA mode would that mean I wouldn't have to change it every time?

**Reply by alex-code** [reply]
You can connect a smartphone to the throttles own WiFi to manage locos so you wouldn't need to change the CS connection settings.

The bottom section **Throttle AP Settings** shows what you'll connect too.

If you can use your CS in STA mode so the throttle and CS are on the same network it'll make it a bit simpler as some smartphones warn about lack of internet and change the WiFi network automatically.
You could then just connect to the throttles IP in a web browser on your home network to manage locos.

Thanks for the appreciation, always nice to know other enthusiasts are enjoying it 👍
